cli/scancommands_test.go

```diff
@@ -0,0 +1,160 @@
+package cli
+
```

Comment:
Move this test to `scans_test.go`

Reply:
this is not full integration test

Reply:
I dont have full integration tests, as the env doesn't spins include curation